tests/php/test-amp-analytics-options.php

```diff
@@ -261,6 +265,8 @@ public function test_amp_print_analytics() {
 
 		$analytics = amp_get_analytics();
 
+		$key = key( $analytics );
+		
```

Comment:
Whitespace here is causing linting error: https://travis-ci.org/ampproject/amp-wp/jobs/593274574#L777-L787